experimental/origin-dapp2/src/pages/App.js

```diff
@@ -1,4 +1,4 @@
-import React, { Component } from 'react'
+import React, { Component, Fragment } from 'react'
```

Comment:
We can use `<></>` instead of `<Fragment></Fragment>` now

Reply:
Yup I've seen that and it is pretty handy. Somehow sublime babel syntax plugin isn't able to handle this and for that reason I didn't use it. Can't find an elegant solution for this